test/JDBC/expected/BABEL-2551.out

```diff
@@ -0,0 +1,102 @@
+create table t1(a int);
+insert t1 values (1), (1), (3), (null);
+GO
+~~ROW COUNT: 4~~
+
+
+create table t2(a int);
+insert into t2 select top(3) a from t1 order by 1;
```

Comment:
add some test cases like : 

insert into select top( <-- using a variable in here ) a from t1 order by 1;

insert into select top with having subquery